monorepo/monorepo-ws/apps/federation-gateway/src/configuration/utils.ts

```diff
@@ -0,0 +1,24 @@
+import { EnumOfErrorTypes, FederationGatewayError } from './errors/errors';
+import { Storage } from '@monorepo-ws/supergraph-manager';
+
+export const validStorage = (storageString: string) => {
+  try {
+    return Storage[storageString];
+  } catch (error) {
+    throw new FederationGatewayError({
+      type: EnumOfErrorTypes.BadStorageType,
+      message: 'Invalid string of storage!',
```

Comment:
do errorů určitě uvádět i co se konkrétně podělalo. uvedl bych sem do payloadu jakej storageString sem přišel, aby bylo vidět proč přesně se to stalo.